### DISCUSSION TITLE
someone just randomly closed my issue when I needed the help with it

### DISCUSSION THREAD

**Discussion by YSPQR** [efroemling/ballistica#328] [General]
whoever did it needs to get punishedd

**Comment by Gengar-lab** [comment]
It was no other than the Creator of the BombSquad himself who closed your issue.

See [this](https://github.com/efroemling/ballistica/issues/326#issuecomment-939157521), he asked you to email him about your issue

**Reply by YSPQR** [reply]
I didn't see no email though thanks

**Reply by efroemling** [reply]
Yes it was me. I prefer to keep issues related to individual accounts off of the forums here and discuss those through direct channels like email. So please email me at support@froemling.net and we can go from there.
